modules/administration-guide/partials/proc_configuring-prod-cli-with-environment-variables-to-use-an-sftp-backup-server.adoc

```diff
@@ -0,0 +1,19 @@
+[id="configuring-prod-cli-for-an-sftp-backup-server-with-environment-variables_{context}"]
+= Configuring {prod-cli} for an SFTP backup server with environment variables
+
+As an alternative to using command-line options for `{prod-cli} server:backup`, you can configure {prod-cli} for an SFTP backup server with environment variables that will allow you to run `{prod-cli} server:backup` without having to enter those arguments as options.
+
+.Prerequisites
+
+* Installed `{prod-cli}`.
+* Configured SFTP backup server. See xref:supported-restic-compatible-backup-servers.adoc[].
+
+.Procedure
+
+* Set the following environment variables:
+** `BACKUP_REPOSITORY_URL` for the backup repository.
+** `BACKUP_REPOSITORY_PASSWORD` for the backup repository password.
+** To log in without a password, set the SSH key using either environment variable:
```

Comment:
While either is a well-known word, do you think we could simplify vocabulary even further and leave "one of the"?

Reply:
Fixed. Thank you!